inc/avatars/namespace.php

```diff
@@ -2,12 +2,13 @@
 
 namespace FAIR\Avatars;
 
+const AVATAR_SRC_SETTING_KEY = 'fair_avatar_source';
+
 /**
  * Bootstrap.
  */
 function bootstrap() {
-	$options = get_option( 'fair_settings', [] );
-	$avatar_source = array_key_exists( 'avatar_source', $options ) ? $options['avatar_source'] : 'fair';
+	$avatar_source = get_site_option( AVATAR_SRC_SETTING_KEY, 'fair' );
```

Comment:
It seems like this is just going to ignore existing settings. That might be fine at this point, with low usage, but I think that single sites should migrate prior settings.

Reply:
i had that thought myself, but is there anything in place for a "run once" update logic?

Reply:
Not yet - but if we're going to change how we do things, we're going to need that. It can just be an activation hook. 

On that note, we also need an uninstall.php, so that we can remove our settings. We overlooked that in the original implementation.

Reply:
let's open a different ticket for the uninstall, but i've got the settings fix included in this PR now.